bindings/java/src/org/sleuthkit/datamodel/CaseDatabaseFactory.java

```diff
@@ -419,6 +422,59 @@ private void createAccountTables(Statement stmt) throws SQLException {
 				+ "FOREIGN KEY(account2_id) REFERENCES accounts(account_id), "
 				+ "FOREIGN KEY(relationship_source_obj_id) REFERENCES tsk_objects(obj_id) ON DELETE CASCADE, "
 				+ "FOREIGN KEY(data_source_obj_id) REFERENCES tsk_objects(obj_id) ON DELETE CASCADE)");
+		
+		stmt.execute("CREATE TABLE tsk_os_account_realms (id " + dbQueryHelper.getPrimaryKey() + " PRIMARY KEY, "
+				+ "name TEXT NOT NULL, "	// realm name - host name or domain name
+				+ "realm_addr TEXT DEFAULT NULL, "		// a sid/uid or some some other identifier, may be null
+				+ "host_id " + dbQueryHelper.getBigIntType() + " DEFAULT NULL, " // if the realm just comprises of a single host
+				+ "name_type INTEGER, "	// indicates if the realm name was  was expressed or inferred 
+				+ "UNIQUE(name, host_id), "
+				+ "FOREIGN KEY(host_id) REFERENCES tsk_hosts(id) )");
+		
+		stmt.execute("CREATE TABLE tsk_os_accounts (os_account_obj_id " + dbQueryHelper.getBigIntType() + " PRIMARY KEY, "
+				+ "login_name TEXT, "	// login name, if available, may be null
+				+ "full_name TEXT DEFAULT NULL, "	// full name, if available, may be null
+				+ "realm_id " + dbQueryHelper.getBigIntType() + ", "		// row id for the realm, may be null if only SID is known 
+				+ "unique_id TEXT, "	// SID/UID, if available
+				+ "signature TEXT NOT NULL, "	// This exists only to prevent duplicates.  It is either �realm_id/unique_id� if unique_id is defined or realm_id/login_name� if unique_id is not defined.
+				+ "status INTEGER, "    // enabled/disabled/deleted
+				+ "admin INTEGER DEFAULT 0," // is admin account
+				+ "type INTEGER, "	// service/interactive
+				+ "created_date " + dbQueryHelper.getBigIntType() + ", "		
+				+ "UNIQUE(signature), "
+				+ "FOREIGN KEY(os_account_obj_id) REFERENCES tsk_objects(obj_id) ON DELETE CASCADE, "
+				+ "FOREIGN KEY(realm_id) REFERENCES tsk_os_account_realms(id) )");
+		
+		stmt.execute("CREATE TABLE tsk_os_account_attributes (id " + dbQueryHelper.getPrimaryKey() + " PRIMARY KEY, "
+				+ "os_account_obj_id " + dbQueryHelper.getBigIntType() + " NOT NULL, "
```

Comment:
I think we should have a "synthetic" PK for this table to put it in 1NF. See discussion in email thread.